thoth/storages/data/alembic/versions/081bee0d9587_create_index_for_thoth_s2i_specific_.py

```diff
@@ -18,11 +18,11 @@
 
 def upgrade():
     # ### commands auto generated by Alembic - please adjust! ###
-    op.create_index("thoth_s2i_image_name", "software_environment", ["thoth_s2i_image_version"], unique=False)
+    op.create_index("thoth_image_name", "software_environment", ["thoth_image_version"], unique=False)
```

Comment:
this file should not be modified, as these files are automatically created! :)

Reply:
Indeed, thanks!